packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -845,24 +1027,23 @@ class IOSDeviceLogReader extends DeviceLogReader {
   /// is true.
   final List<String> _streamFlutterMessages = <String>[];
 
-  /// When using both `idevicesyslog` and `ios-deploy`, exclude logs with the
-  /// "flutter:" prefix if they have already been added to the stream. This is
```

Comment:
what logs have `flutter:` prefix?

Reply:
I'm not entirely sure how it all works, but the Dart VM url will have `flutter:` prefix, any `print` messages from the app will have `flutter:` prefix